internal/stream/stream.go

```diff
@@ -6,18 +6,17 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
-	"net"
 	"net/http"
 	"time"
 
 	"github.com/golang/protobuf/jsonpb"
 	"github.com/opentracing/opentracing-go"
-	"golang.org/x/sync/errgroup"
-
 	"github.com/razorpay/metro/internal/subscriber"
 	"github.com/razorpay/metro/internal/subscription"
+	"github.com/razorpay/metro/pkg/httpclient"
 	"github.com/razorpay/metro/pkg/logger"
 	metrov1 "github.com/razorpay/metro/rpc/proto/v1"
+	"golang.org/x/sync/errgroup"
```

Comment:
nit: move this with the above list of imports

Reply:
done